chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertifiedProductDetailsManagerImpl.java

```diff
@@ -708,7 +708,12 @@ private CertificationResult getCertificationResult(
             final CertifiedProductSearchDetails cp) {
 
         String edition = cp.getCertificationEdition().get("name").toString();
-        Long practiceTypeId = Long.valueOf(cp.getPracticeType().get("id").toString());
+        Long practiceTypeId = null;
+        if (cp.getPracticeType().containsKey("id")) {
+            if (cp.getPracticeType().get("id") instanceof String) {
```

Comment:
Can practiceType (cp.getPracticeType()) ever be null? I wasn't sure exactly which piece of it was null before.

Reply:
I have not seen where it could be, but I did find in the unit testing that cp.getPracticeType().get("id") can be null